src/networking.c

```diff
@@ -3112,6 +3112,10 @@ int canParseCommand(client *c) {
 }
 
 int processInputBuffer(client *c) {
+    mstime_t latency;
+    latencyStartMonitor(latency);
```

Comment:
Has the impact of `gettimeofday` been measured? it will be executed on every single command.

Reply:
I believe `gettimeofday` is a very lightweight operation because of VDSO.